src/github.com/matrix-org/dendrite/common/config/config_test.go

```diff
@@ -59,6 +59,7 @@ listen:
   federation_api: "localhost:7772"
   sync_api: "localhost:7773"
   media_api: "localhost:7774"
```

Comment:
/me wonders why the other components haven't been added here.

Reply:
tracking this via #558